modules/graphql_views/src/Plugin/Deriver/ViewFilterInputDeriver.php

```diff
@@ -58,4 +62,56 @@ public function getDerivativeDefinitions($basePluginDefinition) {
     return parent::getDerivativeDefinitions($basePluginDefinition);
   }
 
+
+  public function isGenericInputFilter($filter) {
+    if (!is_array($filter['value'])) {
+      return false;
+    }
+
+    if (count($filter['value']) == 0) {
+      return false;
+    }
+
+    return is_string( array_keys($filter['value'] )[0] );
```

Comment:
I break it down, from something like that. Because it looks complicated in the first look.

Yes pmelab are right that will work to. But we only need the first key.

```php
$filter['value'] = array();
$filter['value'] = array(
"text",
"test"
);
$filter['value'] = array(
  'distance' => 10,
  'distance2' => 30,
 ...
);
```


```php
// more simpler and combined function
  public function isGenericInputFilter($filter) {
    if (!is_array($filter['value']) &&
        count($filter['value']) == 0
    ) {
      return false;
    }
    $firstKey = array_keys($filter['value'])[0];
    return is_string( $firstKey );
  }
```